### PR TITLE
Story/ece 651 62 menu displaying section UI and frontend

### PR DESCRIPTION
**Pull request by Riya0001**
Test cases added for Menudetails Frontend and UI

Test results location -Documents/General/Sprint 4/System Testing/ECE651-67

### CODE REVIEW

**Review by MareenaFr — COMMENTED** [review]
Unable to access test results document. Please upload the document in MS Teams folder

**Comment by Riya0001** [comment]
Made the element mobile responsive please check

**Review by kar12thik — COMMENTED** [review]
UI looks good, change class to className, for to htmlFor, stroke-linecap to strokeLinecap, stoke-width to strokeWidth, clip-rule to clipRule, fill-rule to fillRule in all places. The ones used are not React attributes. @Riya0001

**Comment by Riya0001** [comment]
Done

File: src/components/SearchFoodOnRestDetailsPage.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        >
+          <svg
+            className="w-5 h-5"
+            fill="none"
+            stroke="currentColor"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              stroke-linecap="round"
```
Comment (kar12thik) [inline src/components/SearchFoodOnRestDetailsPage.js]: Change line 44-46

File: src/components/SearchFoodOnRestDetailsPage.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+function SearchFoodOnRestDetails() {
+  return (
+    <div className="border w-full">
+      <form className="flex items-center w-full">
+        <label for="simple-search" class="sr-only">
```
Comment (kar12thik) [inline src/components/SearchFoodOnRestDetailsPage.js]: Change for, class

File: src/components/SearchFoodOnRestDetailsPage.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+          <div className="absolute inset-y-0 left-0 flex items-center pl-3 pointer-events-none">
+            <svg
+              aria-hidden="true"
+              className="w-5 h-5 text-gray-500"
+              fill="currentColor"
+              viewBox="0 0 20 20"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path
+                fill-rule="evenodd"
```
Comment (kar12thik) [inline src/components/SearchFoodOnRestDetailsPage.js]: Line 18 and 20

**Review by kar12thik — APPROVED** [review]

**Review by MareenaFr — APPROVED** [review]

**Review by db8patel — APPROVED** [review]

**Pull request merged**